packages/core/src/components/graphs/alluvial.ts

```diff
@@ -379,7 +381,9 @@ export class Alluvial extends Component {
 					items: [
 						{
 							label: datum.target.name,
-							value: datum.value + (options.alluvial.units ? ` ${options.alluvial.units}` : ''),
+							value:
+								numberFormatter(datum.value, code) +
+								(options.alluvial.units ? ` ${options.alluvial.units}` : ''),
```

Comment:
let's not do string concatenation here, since a user might return a number through `numberFormatter`